app/src/main/java/org/wikipedia/analytics/SessionFunnel.java

```diff
@@ -104,7 +104,6 @@ private void logSessionData() {
                 "fromExternal", sessionData.getPagesFromExternal(),
                 "fromHistory", sessionData.getPagesFromHistory(),
                 "fromReadingList", sessionData.getPagesFromReadingList(),
-                "fromNearby", sessionData.getPagesFromNearby(),
```

Comment:
Unfortunately, `fromNearby` is a required field in the eventlogging schema. Let's leave this field here, but just give it a value of `0`.

Reply:
Wow! awesome catch! - Done @dbrant